content/concepts/transports/quic.md

```diff
@@ -82,6 +82,14 @@ In addition, a client can make use of QUIC's 0 RTT feature for subsequent connec
 when it has already communicated with a certain server. The client can then send
 (encrypted) application data even before the QUIC handshake has finished.
 
+### QUIC native multiplexing
+
+QUIC interleaves frames from multiple streams into one or more QUIC packets at the
+transport layer. A single QUIC packet can include multiple frames from one or more
+streams. This solves the problem of HOL (head-of-line) blocking: If a packet contain
```

Comment:
```suggestion
A single QUIC packet can include multiple frames from one or more
streams. Since QUIC packets can be decrypted even when they're received out of order, this solves the problem of HOL (head-of-line) blocking: If a packet that contains
```